mkt/site/tests/__init__.py

```diff
@@ -430,6 +430,12 @@ def shortDescription(self):
     def _pre_setup(self):
         super(TestCase, self)._pre_setup()
 
+        # XXX See if we can remove this when we switch to Django 1.8.
+        # Some crud from the migration system gets stuck here and we have to
+        # kick it loose manually.
+        from mkt.webapps.models import AddonUser
+        UserProfile.addonuser_set.related.model = AddonUser
```

Comment:
Did you ever find out the underlying cause? I'd feel better knowing what the actual problem is and possibly fixing it or linking to a Django bug.